test/loadTest/orion_updates_ngsiv2.jmx

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I agree in avoiding duplicates. However, I think the default one should be the simpler case and, in that case, the simpler one is the one not including hw_metrics.

The procedure to enable hw metrics in that file should be included in the README.md file (basically, copy paste of your previous comment in this thread :)